lessons/module-1/js-dot-bracket-notation.md

```diff
@@ -23,6 +21,8 @@ Watch the first 7 minutes of [this video](https://www.youtube.com/watch?v=DJ0dey
 <section class="call-to-action">
 ### Warm Up
 
+Watch the first 7 minutes of [this video](https://www.youtube.com/watch?v=DJ0deyVQZPw) about dot and bracket notation. 
+
```

Comment:
Could you add a markdown comment here briefly stating what the video talks about? Just in case the video disappears from YouTube?

Reply:
That's a great idea Robbie! I'll do that now!